app/steps/ui/starteligibility/template.html

```diff
@@ -0,0 +1,29 @@
+{% extends "includes/layout.html" %}
+{% block page_title %}
+{{ content.title }}
+{% endblock %}
+
+{% block content %}
+
+{% from "widgets/lists.html" import taskAction, unorderedList %}
+
+<main id="content" role="main">
+    {% include "includes/phase_banner_beta.html" %}
+    <div class="grid-row">
+        <div class="column-two-thirds">
+            <h1 class="form-title heading-large">{{content.header}}</h1>
+            <p>{{content.introduction | safe}}</p>
+            <p>{{content.paragraph1 | safe}}</p>
+            <p>{{content.paragraph2 | safe}}</p>
+            {{ unorderedList([content["checklist-item1"], content["checklist-item2"], content["checklist-item3"], content["checklist-item4"], content["checklist-item5"], content["checklist-item6"], content["checklist-item7"]])}}
+            <p>{{content.paragraph3 | safe}}</p>
+        </div>
+    </div>
+    <div class="form-group">
+        <p><a href="/start-apply" class="button button-start">{{common.confirm}}</a></p>
+    </div>
+
+    <p><a href="/tasklist">Return to an existing application.</a></p>
```

Comment:
Can we add this content to the content file?

Also, it doesn't need a full stop at the end